test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -1199,42 +1171,30 @@ export default function Home() {
         ],
       ])
     )
-    const { session, browser } = sandbox
+
+    const { session } = sandbox
 
     await session.assertHasRedbox()
 
-    let stack = next.normalizeTestDirContent(
-      await getRedboxCallStackCollapsed(browser)
-    )
+    const source = await session.getRedboxSource()
+
     if (isTurbopack) {
-      expect(stack).toMatchInlineSnapshot(`
+      expect(source).toMatchInlineSnapshot(`
         "app/utils.ts (1:7) @ [project]/app/utils.ts [app-client] (ecmascript)
-        ---
-        Next.js
-        ---
-        [project]/app/page.js [app-client] (ecmascript)
-        app/page.js (2:1)
-        ---
-        Next.js
-        ---
-        React"
+
+        > 1 | throw new Error('utils error')
+            |       ^
+          2 | export function foo(){}
+          3 |           "
       `)
     } else {
-      expect(stack).toMatchInlineSnapshot(`
+      expect(source).toMatchInlineSnapshot(`
         "app/utils.ts (1:7) @ eval
-        ---
-        ./app/utils.ts
-        file://TEST_DIR/.next/static/chunks/app/page.js (39:1)
-        ---
-        Next.js
-        ---
-        eval
-        ./app/page.js
-        ---
-        ./app/page.js
-        file://TEST_DIR/.next/static/chunks/app/page.js (28:1)
-        ---
-        Next.js"
+
+        > 1 | throw new Error('utils error')
```

Comment:
The other frames are ignored by default, so we only need to check the source. And the group `Next.js` and `React` are gone now

Reply:
> The other frames are ignored by default, so we only need to check the source.

That's a misunderstanding around what tests are for. If the other frames are ignored, assert on it. Otherwise the test doesn't actually test it. It just hopes that the implementation continues to work.